### DISCUSSION TITLE
How to change issue number? #?

### DISCUSSION THREAD

**Discussion by R3MIGIJUS** [community/community#23580] [Projects and Issues]
<p>Is it possible to change number of issue? Like from <span class="hashtag">#15</span> to <span class="hashtag">#1</span>? In my project i was deleted all old issue and started to create new, but issue number is continuos, doesn’t start from <span class="hashtag">#1</span>.</p>
<p>Thank you.</p>

**Comment by Katsute** [comment, marked as answer]
<p>You aren’t allowed to change issue numbers, it would cause links referring to old issues to be broken. The only way to reset the issue counter would be to create an entirely new repository.</p>
